app/models/mortality_tracking.rb

```diff
@@ -43,4 +43,38 @@ class MortalityTracking < ApplicationRecord
     COMMENTS: "Comments"
   }
 
+  def self.create_from_csv_data(attrs)
+    attrs['shl_case_number']                     = attrs.delete('shl_number')
+    attrs['number_morts']                     = attrs.delete('_morts')
+    new(attrs)
+  end
+
+  validates_inclusion_of :raw, in: [true, false]
+
+  validates_presence_of :mortality_date
+  # mortality_date has 'Unknown' and 'unkown' in csv
+  validates_presence_of :cohort
+  validates_presence_of :shl_case_number
+  validates_presence_of :spawning_date
+  validates_presence_of :number_morts
+
+  validates :number_morts, numericality: true, allow_blank: true
+  # number_morts has been '9+' and 'TBD' in csv
+
+  validates :processed_by_shl, format: { with: /Y|N/ }, allow_blank: true
+
+  def mortality_date=(mortality_date_str)
+    return unless mortality_date_str
+    write_attribute(:mortality_date, DateTime.strptime(mortality_date_str, '%m/%d/%y'))
+  rescue
+    write_attribute(:mortality_date, nil)
+  end
+
+  def spawning_date=(spawning_date_str)
+    return unless spawning_date_str
+    write_attribute(:spawning_date, DateTime.strptime(spawning_date_str, '%m/%d/%y'))
```

Comment:
@edwinthinks what would your thoughts be on using `DateTime#parse` instead here to provide more flexibility in formatting?

Reply:
I don't mind using `parse`. However, I wonder if we gain some precision by specifying the format of the time? Fine either way :)

Reply:
Ahh super interesting. I didn't realize this but `DateTime.parse` has an issue with the date '4/23/01'. Aka
```
DateTime.parse('4/23/01')
```
will raise an error. Going to continue using the original method.